build.gradle

```diff
@@ -455,4 +392,70 @@ installDist.dependsOn downloadGsaLibFile, downloadIntelDeflater
 build.dependsOn installDist
 check.dependsOn installDist
 
+model {
```

Comment:
This had to move lower because the shadow and sparkJar tasks weren't defined yet where it was initially.  It would be better to put the dependOn block in them, but I can't figure out how to reference the tasks defined in the model, I'm sure it's doable, but I can't figure out the magic invocation needed.